### ISSUE TITLE
Tabbed Layout

### ISSUE THREAD

**Issue by ezzobirbezziou** [leftwm/leftwm#1248]
### Describe the feature

I would like to have tabbed layout like i3wm and herbsluftwm

### How important is this feature to you?

Would be nice to have

### How important do you think is this feature for the average LeftWM user?

No one uses LeftWM because this is missing

**Comment by alexanderwiller** [comment]
I'd like to reinforce the need for such a feature. In the past, I've been using i3wm and tabbed layout was a very typical arrangement for windows in my use case.

That use case often included multiple instances of a multi-window program, e.g. KiCad, with up to five windows used regularly (Project manager, footprint library, schematic library, schematic editor, board editor).
  
All these windows belong together, and multiple projects (realistically three or four or so) can be open at the same time, leading to up to 20 windows that can easily be confused if not arranged cleanly.

I've then allocated one workspace for each project, with all the windows to that project being in a tabbed layout.

I don't know how this use case can be supported with another method than tabs, but am open to any ideas.

**Comment by AnthonyMichaelTDM** [comment]
I've never used i3wm, but wouldn't the existing tags functionality provide a similar experience to tabs?

**Comment by hertg** [comment]
@yitsushi I'm also unfamiliar with i3 and the tabbed layout, but what you are describing sounds fairly similar to our `MainAndDeck` layout. We don't show a "tab list" though.

**Comment by yitsushi** [comment]
> @yitsushi I'm also unfamiliar with i3 and the tabbed layout, but what you are describing sounds fairly similar to our `MainAndDeck` layout. We don't show a "tab list" though.

Yes, it is. Probably if it was documented better, it would even resolve this at least partially. But last time I decided to try it, I couldn't even figure out how to navigate between items in the deck. At the end I decided to live without it and don't spend more time on it to figure out how can I use it effectively. So probably most cases can be covered by documentation (for example I don't really care about the "tab list").

For full support it should support all kind of variations and options. Now MainAndDeck is handled in a special way. But it could support 3 column deck too, like "one small on the left, one big in the middle, and smaller on the right and the right one is a deck" too.
